plugin/src/main/java/com/denizenscript/denizen/events/entity/ProjectileHitsScriptEvent.java

```diff
@@ -52,12 +55,65 @@
     //
     // @Location true
     //
-    // @Triggers when a projectile shot by an entity hits a block.
+    // @Cancellable true
+    //
+    // @Triggers when a projectile shot by an entity hits a block or a fish hook lands on a block.
     //
     // @Context
     // <context.projectile> returns the EntityTag of the projectile.
     // <context.shooter> returns the EntityTag of the shooter, if there is one.
     // <context.location> returns the LocationTag of the block that was hit.
+    // <context.hit_face> returns a LocationTag vector of the hit normal (like '0,1,0' if the projectile hit the top of the block).
+    //
+    // -->
+
+    // <--[event]
+    // @Events

```

Comment:
This should probably be made as a new event, rather than having now 4 documented events all in one class file